### DISCUSSION TITLE
Internal Error

### DISCUSSION THREAD

**Discussion by Bhumika1997** [chaos-mesh/chaos-mesh#2059] [Q&A]
Hi,
while performing network experiments i am getting an error

An error occurred: Internal error occurred: failed calling webhook "vpodnetworkchaos.kb.io": Post "https://chaos-mesh-controller-manager.chaos-testing.svc:443/validate-chaos-mesh-org-v1alpha1-podnetworkchaos?timeout=5s": context deadline exceeded

Can anybody help me on this?

**Comment by directtv999** [comment]
are you able to solve this issue?
